File: URI_1047_game_time_with_minutes.js
const lines = ["7 10 8 9"];

const inputArr = lines[0].split(" ");
const MINUTES_PER_HOUR = 60;
const startMinute = Number(inputArr[0]) * MINUTES_PER_HOUR + Number(inputArr[1]);
const endMinute = Number(inputArr[2]) * MINUTES_PER_HOUR + Number(inputArr[3]);

let totalMinutes = 0;
if (startMinute < endMinute) {
    totalMinutes = endMinute - startMinute;
} else if (startMinute >= endMinute) {
    totalMinutes = endMinute + 24 * MINUTES_PER_HOUR - startMinute;
}

let totalHours = Math.floor(totalMinutes / MINUTES_PER_HOUR);
totalMinutes = totalMinutes % MINUTES_PER_HOUR;
console.log("O JOGO DUROU " + totalHours + " HORA(S) E " + totalMinutes + " MINUTO(S)");

File: URI_1047_game_time_with_minutes.java
import java.util.Scanner;

public class URI_1047_game_time_with_minutes {

	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		final int MINUTES_PER_HOUR = 60;
		int startMinute = scanner.nextInt() * MINUTES_PER_HOUR + scanner.nextInt();
		int endMinute = scanner.nextInt() * MINUTES_PER_HOUR + scanner.nextInt();

		int totalMinutes = 0;
		if (startMinute < endMinute) {
			totalMinutes = endMinute - startMinute;
		} else if (startMinute >= endMinute) {
			totalMinutes = endMinute + 24 * MINUTES_PER_HOUR - startMinute;
		}

		int totalHours = totalMinutes / MINUTES_PER_HOUR;
		totalMinutes = totalMinutes % MINUTES_PER_HOUR;
		System.out.println("O JOGO DUROU " + totalHours + " HORA(S) E " + totalMinutes + " MINUTO(S)");

		scanner.close();
	}
}